src/winetricks

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
 
```

Comment:
I'm not too familiar with 'vim', but from a quick look-see, I find more suggestions to having it in the end of the file instead of the beginning, although over at 'vim.wikia.com' [here] (http://vim.wikia.com/wiki/Modeline_magic#Examples), they say it can be up or down.

Aside from the few findings, I don't think I've ever seen it myself before, anywhere, so I can't say which really is the more common practice.  ^^